scripts/generate_fine_tune_data.py

```diff
@@ -0,0 +1,51 @@
+# bad name
```

Comment:
This comment should probably be changed to clarify what you mean or to remove it fully